README.md

```diff
@@ -0,0 +1,155 @@
+# [drf-paginator](https://github.com/yola/drf-paginator)
+
+JavaScript module for consuming paginated Django REST framework endpoints.
+
+[![npm Version](https://img.shields.io/npm/v/drf-paginator.svg?style=flat-square)](https://www.npmjs.com/package/drf-paginator)
+[![MIT License](https://img.shields.io/badge/license-MIT-blue.svg?style=flat-square)](https://github.com/yola/drf-paginator/master/LICENSE)
+[![Build Status](https://img.shields.io/travis/yola/drf-paginator.svg?style=flat-square)](https://travis-ci.org/yola/drf-paginator)
+[![Dependencies Status](https://img.shields.io/david/yola/drf-paginator.svg?style=flat-square)](https://www.npmjs.com/package/drf-paginator)
+[![devDependencies Status](https://img.shields.io/david/dev/yola/drf-paginator.svg?style=flat-square)](https://www.npmjs.com/package/drf-paginator)
+
+## Examples
+
+### Creating a paginator and fetching the first page:
+
+```javascript
+import drfp from 'drf-paginator';
+
+
+const paginator = drfp.paginate(request);
+
+paginator.first()
+  .then(function(response) {
+     // Do something with the response
+  });
+```
+
+### Creating a paginator using limit-offset pagination style:
+
+```javascript
+import {Paginator} from 'drf-paginator';
+
+
+const limit = 50;
+const style = Paginator.styles.limitOffset;
```

Comment:
The `style` option determines what `queryParams` are sent to the request. Requests aren't supposed to interpret the query params, but only serialize them. This way the paginator is configured to match the request rather than the other way around.